### DISCUSSION TITLE
Faculdade: IMPACTA ou University of People?

### DISCUSSION THREAD

**Discussion by anavitoria369** [frontendbr/forum#1632] [Discussão]
Farei faculdade em 2020 e ainda tenho muitas dúvidas sobre qual escolher. 
Como eu moro no Norte do Brasil, e as melhores opções de faculdade ficam no sudeste e sul, pretendo cursar ADS á distância. Porém, fiquei em dúvida entre IMPACTA e UoP, por conta de suas grades curriculares muito boas. 

Alguém aqui já estudou nessas faculdades? Como foi a experiência? Recomendam? 
Obs.: gostaria de focar na Programação.

**Comment by zmmarina** [comment]
@anavitoria369 Ana, tudo bem? Você se decidiu por alguma instituição dessas q falou?
Estou pensando em me matricular na UoP, mas tenho medo de ser uma fraude. Tipo.. o diploma não ser aceito ou esse tipo de coisa. Se vc puder, compartilha sua experiência no tema. Valeu!

**Comment by tbooom** [comment]
Estudei na faculdade impacta ( de forma presencial ) e não recomendo.
Acredito que o melhor é estudar programação comprando cursos voltados ao que curte, e depois que fizer algo para o portifólio entre em uma facul.

**Comment by gersoncjlima** [comment]
Oi Ana, vc mora aqui no norte, qual a cidade. Tenta fazer o IFPA, caso você
more aqui em Belém.

Atenciosamente,

Gerson Lima






Em qui., 9 de jul. de 2020 às 19:56, tbooom <notifications@github.com>
escreveu:
